### PR TITLE
Update docs for "inputs" -> "input"

### PR DESCRIPTION
**Pull request by dwhitena**
Updates for the high priority docs to be consistent with the change from "inputs" to "input".  We need to follow up with the fixes for the other examples (wordcount and ML).

### CODE REVIEW

File: doc/fundamentals/distributed_computing.md [hunk 1]
```diff
@@ -30,41 +30,68 @@ If you use the `CONSTANT` strategy, Pachyderm will start the number of workers t
...
 
 ```
-"inputs": [
-    {
-      "repo": {
-        "name": "string"
-      },
-      "glob": "string"
+"input": {
+    "atom": {
```
Comment (derekchiang) [inline doc/fundamentals/distributed_computing.md]: This seems somewhat confusing.  Can you somehow make it more explicit that you can't actually have more than one input types at the top level?

File: doc/reference/pipeline_spec.md [hunk 2]
```diff
@@ -35,7 +35,15 @@ create-pipeline](../pachctl/pachctl_create-pipeline.html) doc.
     "cpu": double
   },
   "input": {
```
Comment (derekchiang) [inline doc/reference/pipeline_spec.md]: Same comment: you can't actually have more than one input types at the top level.  Has to be one of `atom/union/cross`.

File: doc/fundamentals/distributed_computing.md (hunk 1 above)
Comment (dwhitena) [inline doc/fundamentals/distributed_computing.md]: Fixed.

File: doc/reference/pipeline_spec.md (hunk 2 above)
Comment (dwhitena) [inline doc/reference/pipeline_spec.md]: Fixed. Thanks.

File: doc/reference/pipeline_spec.md [hunk 3]
```diff
@@ -250,12 +283,9 @@ the commit itself) will be processed.
 
 Union inputs take the union of other inputs. For example:
 
-| inputA | inputB | inputA ∪ inputB |
-| ------ | ------ | --------------- |
-| foo    | fizz   | foo             |
-| bar    | buzz   | fizz            |
-|        |        | bar             |
-|        |        | buzz            |
+- inputA: foo, bar
```
Comment (derekchiang) [inline doc/reference/pipeline_spec.md]: Why this change?  I personally like the original formatting better.

File: doc/reference/pipeline_spec.md [hunk 4]
```diff
@@ -269,12 +299,9 @@ reason to take a union of unions since union is associative.
 Cross inputs take the cross product of other inputs, in other words it creates
 tuples of the datums in the inputs. For example:
 
-| inputA | inputB | inputA ⨯ inputB |
-| ------ | ------ | --------------- |
-| foo    | fizz   | (foo, fizz)     |
-| bar    | buzz   | (foo, buzz)     |
-|        |        | (bar, fizz)     |
-|        |        | (bar, buzz)     |
+- inputA: foo, bar
```
Comment (derekchiang) [inline doc/reference/pipeline_spec.md]: Same here.

File: doc/reference/pipeline_spec.md (hunk 3 above)
Comment (dwhitena) [inline doc/reference/pipeline_spec.md]: I did too. Unfortunately it doesn't render correctly in HTML.  You can check the current version to see or run `make html` from `docs`

File: doc/reference/pipeline_spec.md (hunk 4 above)
Comment (dwhitena) [inline doc/reference/pipeline_spec.md]: See above.
